docs/pages/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -0,0 +1,622 @@
+
+When a user requests access to a resource ID, the Teleport Auth Service does the
+following:
+
+1. Collects all the roles named in `allow.request.search_as_roles`, filtering
+   these to exclude roles specified in `deny.request.search_as_roles` or
+   `deny.request.roles`. 
+1. Determines which of the remaining roles can access the requested resource.
+   For a Resource Access Request to be valid, one of the role matchers listed in
+   a user's `search_as_roles` configuration must match a role that permits
+   access to the requested resources.
+
+## How long access lasts
+
+You can configure the length of time that Teleport will grant a user elevated
+privileges for an approved Access Request.
+
+### The `request.max_duration` field
+
+The `max_duration` option indicates the maximum length of time that a user is
+allowed to have elevated privileges for particular roles. You can specify it
+with a role like the following:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: temp-dba
+spec:
+  allow:
+    request:
+      # Allow access to role `dba` for up to 4 days.
+      roles: ['dba']
+      max_duration: 4d
+```
+
+The value of `max_duration` can never exceed seven days.
+
+### Calculating the duration of elevated privileges
+
+When Teleport grants a user's Access Request, the `max_duration` field is only
+one factor in determining how long the user's access will last. Teleport uses
+the following logic to make this calculation, which never exceeds
+`max_duration`: 
+
+1. Calculate the maximum duration of elevated privileges if the Access Request
+   were granted. This is the lowest value of:
+    - The `--max-duration` flag of the [`tsh request
+      create`](../../reference/cli/tsh.mdx#tsh-request-create) command (if the
+      user creating the request provides this flag).
+    - The lowest value of a `request.max_duration` field that corresponds to one
+      of the user's requested roles. 
+
+1. Calculate the session TTL of the certificate the user would receive if
+   Teleport were to grant the Access Request. This calculation chooses the
+   lowest value of: 
+   - The requested session expiration time, which is the value of the
+     `--session-ttl` flag of [`tsh request
+     create`](../../reference/cli/tsh.mdx#tsh-request-create).
+   - The remaining time in the user's current Teleport session.
+   - The lowest value of the `options.max_session_ttl` field in the user's
+     requested roles.
+
+1. If the maximum duration is greater than zero, set the duration of elevated
+   privileges to either the maximum duration or the session TTL calculated
+   earlier, whichever is shorter. Otherwise, set the duration of elevated
+   privileges to the session TTL.
+
+### How long Access Requests are valid
+
+Teleport uses the following logic to determine how long an Access Request is
+valid while it awaits approval:
+
+1. Begin with with the base expiration of the Access Request, which a user can
```

Comment:
I believe @atburke _just_ added this to the web UI too (a day or two ago - may not even be released yet)